src/java.base/linux/classes/jdk/internal/platform/CgroupSubsystemController.java

```diff
@@ -25,11 +26,12 @@
 
 package jdk.internal.platform;
 
+import java.io.BufferedReader;
```

Comment:
Unused now?

Reply:
Yes, it's now unused. I've removed it in the latest update to the PR and also replaced a few more Paths.get() with Path.of() in the current set of files that have been updated in this PR.